payments-core/src/main/java/com/stripe/android/view/StripeEditText.kt

```diff
@@ -96,6 +96,7 @@ open class StripeEditText @JvmOverloads constructor(
         }
 
     init {
+        textWatchers = mutableListOf()
```

Comment:
It might be worth a quick check of CardMultiLineWidget and CardFormView, they also set the listeners.   I don't see any reason why this wouldn't fix it though.

Reply:
Yes, it works.